src/components/language/language.repository.ts

```diff
@@ -111,6 +111,27 @@ export class LanguageRepository extends DtoRepository(Language) {
       .first();
   }
 
+  async approvedInventory(language: Language) {
+    return await this.db
+      .query()
+      .match([
+        node('lang', 'Language', { id: language.id }),
+        relation('in', '', 'language', { active: true }),
+        node('engagement', 'LanguageEngagement'),
+        relation('in', '', 'engagement', { active: true }),
+        node('project', 'Project'),
+        relation('out', '', 'approvedInventory', { active: true }),
+        node('prop', 'Property'),
+      ])
+      .where({
+        prop: {
+          value: true,
+        },
+      })
+      .return('project')
```

Comment:
This could be done in the Language.hydrate method and we could save the separate lookup. I think it would be faster.
```ts
approvedInventory: `exists((node)
  <-[:language { active: true }]-(:LanguageEngagement)
  <-[:engagement { active: true }]-(:Project)
  -[:approvedInventory { active: true }]->(:Property { active: true })
)`
```